packages/base/src/Runtimes.js

```diff
@@ -22,7 +21,6 @@ const registerCurrentRuntime = () => {
 		currentRuntimeIndex = Runtimes.length;
 		Runtimes.push({
 			...VersionInfo,
-			url: metaUrl,
```

Comment:
Please also remove the usage of this in `CustomElementsRegistry.js` line 51

```js
	if (allRuntimes.length > 1) {
		message = `${message}\nLoading order (versions before 1.1.0 not listed): ${allRuntimes.map(runtime => `\n${runtime.description} ${runtime.url}`).join("")}`;
	}
```

Reply:
done